test/test_pipeline_extractive_qa.py

```diff
@@ -100,17 +100,22 @@ def test_extractive_qa_answers_with_translator(
         prediction["answers"][0].context == "My name is Carla and I live in Berlin"
     )
 
-
-@pytest.mark.parametrize("retriever_with_docs", ["tfidf"], indirect=True)
-@pytest.mark.parametrize("document_store_with_docs", ["elasticsearch"], indirect=True)
-def test_extractive_qa_eval(reader, retriever_with_docs, tmp_path):
-    queries = ["Who lives in Berlin?"]
-    labels = [
+EVAL_QUERIES = ["Who lives in Berlin?", "Who lives in Munich?"]
+EVAL_LABELS = [
         MultiLabel(labels=[Label(query="Who lives in Berlin?", answer=Answer(answer="Carla", offsets_in_context=[Span(11, 16)]), 
             document=Document(id='a0747b83aea0b60c4b114b15476dd32d', content_type="text", content='My name is Carla and I live in Berlin'), 
+            is_correct_answer=True, is_correct_document=True, origin="gold-label")]),
+        MultiLabel(labels=[Label(query="Who lives in Munich?", answer=Answer(answer="Carla", offsets_in_context=[Span(11, 16)]), 
+            document=Document(id='something_else', content_type="text", content='My name is Carla and I live in Munich'), 
             is_correct_answer=True, is_correct_document=True, origin="gold-label")])
     ]
 
+@pytest.mark.parametrize("retriever_with_docs", ["tfidf"], indirect=True)
+@pytest.mark.parametrize("document_store_with_docs", ["elasticsearch"], indirect=True)
+def test_extractive_qa_eval(reader, retriever_with_docs, tmp_path):
```

Comment:
I'd expect these tests rather in `test_eval.py` :thinking: 
Especially when we have some "shared function" that are not only relevant for extractive qa (e.g. eval of retrieval pipelines, saving/loading of results ...)

What do you think?

Reply:
done: added tests for all query digesting standard pipelines, too.